JesusFilm/core
# feat: add rtl to core from material ui

# Description

Implement RTL functionality from [Material UI](https://mui.com/material-ui/guides/right-to-left/) to our Journey project. 

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413044377)

# How should this PR be QA Tested?

_Preconditions_

1. Create a journey 
2. Update the journey language to one of the supported RTL languages via admin
3. Append the journeyId to the end of this URL: https://journeys-git-22-08-jg-feat-add-rtl-material-ui-jesusfilm.vercel.app/
4. Visit the URL: https://journeys-git-22-08-jg-feat-add-rtl-material-ui-jesusfilm.vercel.app/{journeyId}


- [x] it should display right aligned text when journey has rtl locale
- [x] it should display left aligned text when journey has ltr locale
- [x] it should set html dir as "rtl" when journey has rtl locale

# Checklist
